### DISCUSSION TITLE
CONGRATULATIONS!!! 360º Image Viewer using jsPanel

### DISCUSSION THREAD

**Discussion by inglesuniversal** [Flyer53/jsPanel4#157] [Ideas]
How can I make my own 360 image viewer based on your code example...

https://alpha.jspanel.de/docs/demos/test-hugo.html

PS. Does your brother offer free access to the 360º images he has taken before or do you have a source page that make offer free downloads for that matter.

Our site is focused on English Learning Students.

**Comment by Flyer53** [comment]
Hi Hugo,

That very much depends on what kind of image you want to display in what manner. Is it just a plain rectangular panorama (basically an image like any other), a 360° image, a sperical panorama, does it have several levels to zoom in (each level showing more details), and so on .... In any case those panoramas have to be photographed very carefully in a specific manner. Each panorama consists of a collection of at least 6 images, sometimes hundreds or even thousands of individual images that have to be processed in a pretty time-consuming process in order to get a good result.
Then you need an image viewer capable to display the panorama. My brothers website uses [krpano.js](https://krpano.com/home/) which requires a license and is not free. There are one or two other js viewers that can handle sophisticated panorama images. I never tried anything else than krpano.js which offers a tremenduous amount of features but is quite complicated to configure.

My brother does those panorama images for a living. Depending on the purpose and required result it sometimes takes weeks to complete a single sperical panorama. So no, they are not free. What exactly would you want to use it for? Just a single image or more? Is the site commercial or free to use for everyone?

Regards,
Stefan

**Comment by inglesuniversal** [comment]
The idea is to implement panoramas on my website for our ESL students so they can describe the surroundings of the images, describe the place, weather, objects and so forth. Let me see if I can find any resources on github and if I do ... I will gladly share them with you as well. Good luck!

**Comment by Flyer53** [comment]
@inglesuniversal 
Any news on this issue Hugo?
